auacm/app/templates/index.html

```diff
@@ -90,5 +90,6 @@
   <script src="/static/js/controllers/SettingsController.js"></script>
   <script src="/static/js/controllers/ViewProblemController.js"></script>
   <script src="/static/js/controllers/EditProblemController.js"></script>
+  <script src="/static/js/controllers/EditCompetitionController.js"></script>
```

Comment:
I didn't see a bunch of duplicate code, but I thought I should bring it up: would it be wise to try to combine the CreateCompetitionController and EditCompetitionController like we did with problems? Again, I'm not advocating for it, I just wanted to throw it out there.